Kernel/OperatorType.hpp

```diff
@@ -94,6 +94,11 @@ class OperatorType
 
   static OperatorType* getTypeFromKey(OperatorKey* key, unsigned taArity);
 
+  static TermList getEmpty() { 
+    static TermList empty(0,false);
```

Comment:
I find the static here a bit confusing, since we are returning by value. But it will work, so no big deal...

Reply:
Yes - also I keep wanting a constructor for this in `TermList`, maybe it can go there? But also please kill the `static`, this `TermList empty; empty.MakeEmpty(); return empty;` should be inlined and optimised to "move bytes representing empty-term into some destination" which is what we want.

Reply:
Right, good point! I knew I was doing something silly here.